src/meta/src/controller/streaming_job.rs

```diff
@@ -784,39 +773,12 @@ impl CatalogController {
     pub async fn post_collect_job_fragments(
         &self,
         job_id: ObjectId,
-        actor_ids: Vec<crate::model::ActorId>,
         upstream_fragment_new_downstreams: &FragmentDownstreamRelation,
-        split_assignment: &SplitAssignment,
         new_sink_downstream: Option<FragmentDownstreamRelation>,
     ) -> MetaResult<()> {
         let inner = self.inner.write().await;
         let txn = inner.db.begin().await?;
 
-        let actor_ids = actor_ids.into_iter().map(|id| id as ActorId).collect_vec();
```

Comment:
Where do we update the FragmentSplits table for caller on this method?

Reply:
https://github.com/risingwavelabs/risingwave/blob/d9857a01305f08edadd474819925dc5ecba3fd60/src/meta/src/barrier/context/context_impl.rs#L421

All the changes for the source manager are placed outside. Do you want me to move them inside this?